plugins/inputs/intel_rdt/intel_rdt.go

```diff
@@ -497,7 +497,7 @@ func validateInterval(interval int32) error {
 func splitMeasurementLine(line string) ([]string, error) {
 	values := strings.Split(line, ",")
 	if len(values) < 8 {
-		return nil, fmt.Errorf(fmt.Sprintf("not valid line format from pqos: %s", values))
+		return nil, fmt.Errorf("not valid line format from pqos: %q", values)
```

Comment:
We are not currently adding quotes to this string. Not fond of adding quotes around something after a colon either. It can lead to confusion about whether those quotes are actually in that value or not.

Can this stay a `%s`?

Reply:
Changed to `%s`